about.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: "About the project"
+category: Project
+---
+RSE Competencies Toolkit is a resource to support RSEs (Research Software Engineers) in tracking and managing their professional development. It is currently in the early stages of development and we welcome contributions and feedback.
+
+Note that, while we've focussed on RSE during the early stages of development, this tool can be used for any roles. We plan to extend this to other research roles and welcome contributions and collaboration from anyone working in research, broadly defined.
+
+RSE Competencies Toolkit comprises:
+
+* An RSE competency framework, outlining a structured set of skills that are useful when working as an RSE, with examples of how these skills can be demonstrated at different levels of experience. Not all RSEs will or need to have all skills at all levels.
+* A curated set of training resources, linked to the skills and levels from the competency framework.
+* A tool to visualise and compare different competency profiles.
+
+RSE Competencies Toolkit aims to support the following uses:
+
+* Recording and visualising your competency profile as an individual RSE
+* Comparing competency profiles across a group of RSEs (e.g. to show the commonalities and variety across RSEs doing the same role at the same level at the same organisation, or comparing across organisations)
+* Find high-quality training resources to improve skills in a particular competency
+* Define aspirational competency profiles, illustrate the gap to your current profile and highlight training resources that could help bridge that gap.
+
+### Get in touch
+
+* Join our Slack channel in the UK RSE Workspace ([#rse-competencies-toolkit](https://ukrse.slack.com/archives/C05CY0YFWEL))
+* Join our Google Groups group ([groups.google.com/g/rse-competencies-toolkit](https://groups.google.com/g/rse-competencies-toolkit))
+* Star us on Github ([github.com/RSEToolkit/rse-competencies-toolkit](https://github.com/RSEToolkit/rse-competencies-toolkit/issues))
+
+### Support the project
+
+If you would like to contribute, please look at the list of [Issues on the Github repository](https://github.com/RSEToolkit/rse-competencies-toolkit/issues) for ways to help. 
```

Comment:
Could also link to https://github.com/RSEToolkit/rse-competencies-toolkit/tree/main#support-needed